tests/test_level/test_level_01.py

```diff
@@ -481,10 +482,17 @@ def test_turn_left_nl(self):
 
     def test_turn_with_text_gives_error(self):
         code = "turn koekoek"
+        expected = "pass"
+
+        skipped_mappings = [
```

Comment:
I don't think all tests need more explanation but these numbers seems quite magical at first glance. Maybe explain 1 test (the first one) in words?

Reply:
Sure thing! [Improve explanation/naming of testing & SourceRange](https://github.com/hedyorg/hedy/pull/4320/commits/066531d23c96e9862362927cfc2b58866d52db31) should resolve this and improve the explanation of the source range as well.